src/osemgrep/core/Cli_match.ml

```diff
@@ -0,0 +1,99 @@
+module OutT = Semgrep_output_v1_t
+open Common
+
+let name (c : OutT.cli_match) =
```

Comment:
small comment explaining what this helper does.

Reply:
done in [a7869a2](https://github.com/semgrep/semgrep/pull/9706/commits/a7869a28d7983febd689530478e839c9d02c6e5c)